### ISSUE TITLE
question about the code

### ISSUE THREAD

**Issue by TqXue** [dbp1994/bare-wacv-2023#4]
Good works! 
But i am confusing that in losses.py line 216  where  `weighted_loss = F.cross_entropy(prediction, target_label)` .
I know that you want to just selece the 'trustworthy' samples in each mini-batch to caulate the loss and update the network, but if there are none 'trustworthy' samples in a mini-batch, the code will run the line 216, which means just use these 'untrustworthy' samples to caulate the loss and update the network ? ? It seems inappropriate.

**Comment by dbp1994** [comment]
You are right in that we could select examples that are actually 'untrustworthy' but the strength of the algorithm lies in the following: We are selecting samples only at the level of each mini-batch. We may or may not have made a mistake in selecting samples. However, the same, rejected samples could very well be selected in the next epoch of training. 

I hope this helps!

P.S: Sorry for the late reply!
